documentstore/restfulapi.py

```diff
@@ -271,6 +291,25 @@ def fetch_documents_bundle(request):
         return HTTPNotFound(str(exc))
 
 
+@bundles.put(
+    schema=DocumentsBundleSchema(),
+    validators=(colander_body_validator,),
+    accept="application/json",
+    renderer="json",
+)
+def put_documents_bundle(request):
+    try:
+        request.services["create_documents_bundle"](
+            request.matchdict["bundle_id"], metadata=request.validated
+        )
+    except KeyError:
```

Comment:
A exceção `KeyError` para `request.matchdict["bundle_id"]` nunca será lançada se a função for invocada pelo Pyramid.